### PR TITLE
Remove occupies predicates and shorten isGeographicallyContainedIn 

### LINKED ISSUE #809: Remove `occupiesGeographically` and `occupiesGeographicallyPermanently`?
During the inverse working group meeting, we noticed a potential redundancy with the following predicates.

- Currently, we have `isGeographicallyOccupiedBy` and its inverse, `occupiesGeographically`. We also have `isGeographicallyPermanentlyOccupiedBy` and its inverse, `occupiesGeographicallyPermanently`.
- While not formally specified, these predicates are likely intended to relate objects to spatial regions. (`isGeographicallyContainedIn`, a further property not on the list above, is likely intended to relate spatial regions to spatial regions.)
- Arguably, `hasPhysicalLocation` could do the work that `occupiesGeographically` does, and for this reason we may want to remove `occupiesGeographically` and its inverse to avoid redundancy.
- If we were to remove `occupiesGeographicallyPermanently` and its inverse, there is a question as to whether we would want to add an analogous `hasPermanentPhysicalLocation`.

### LINKED ISSUE #812: Shorten geo predicates
Some of us think the shift to more explicit, but longer, predicates in gist 11 was overkill. The proposal is to go back to something like (without inverses, which are eliminated in #506):

* geoContains
* geoOccupies
* geoOccupiesPermanently 

However, the occupies predicates are considered for removal in issue #809.

### PR DESCRIPTION
**Pull request by rjyounes**
Fixes #809 - see https://github.com/semanticarts/gist/issues/947#issuecomment-1679565100
Fixes #812 

NB: I've deliberately kept the full prefLabel "is geographically contained in" for `isGeoContainedIn`. This is a bending of our rule about constructing pref labels, but in this case is warranted, in my opinion. I've updated the style guide to reflect this change.

### CODE REVIEW

File: docs/gistStyleGuide.md [hunk 1]
```diff
@@ -130,7 +131,14 @@ Note: As of version 12.0.0, gist itself does not itself follow the infix convent
...
 - Normalized to natural language standards. E.g., hyphens inserted, acronyms in all caps, proper nouns capitalized, etc.
-- Examples: *has unit of measure*, *has SSN*, *unit symbol Unicode*
+- Examples: *has unit of measure*, *has SSN*
+
+### Valid Exceptions
+
+There may occasionally be valid reasons to deviate from the conventions stated here:
+
+- Deviation from wording of the local name. For example, the predicate `gist:isGeoContainedIn` uses a shortened form of "geographically" for conciseness. The `skos:prefLabel` uses the fully spelled out word: "is geographically contained in."
+- Deviation from typographical conventions such as case. For example, a proper name in a property label could appropriately be capitalized. Since this will fail the SHACL validation constraints during the ontology build and release process, add the term to the `gistValidationAnnotations.ttl` file so that label validation will be skipped.
```
Comment (kchastain703) [inline docs/gistStyleGuide.md:141]: Should we give an example of this, too?

**Review by uscholdm — APPROVED** [review]
Looks good.

File: docs/gistStyleGuide.md (hunk 1 above)
Comment (dylan-sa) [reply docs/gistStyleGuide.md:141]: I agree w/ @kchastain703 that an example would be helpful.

Comment (dylan-sa) [inline docs/gistStyleGuide.md:141]: In the 'Properties' section above, it already says we can deviate from lower case for proper nouns. So the exception here doesn't seem like a genuine exception to the convention as stated. Maybe we can say the convention is simply lowercase, and then the bit about normalizing to natural language standards can be left to the 'Valid Exceptions' section?

**Review by dylan-sa — CHANGES_REQUESTED** [review]

File: docs/gistStyleGuide.md (hunk 1 above)
Comment (rjyounes) [reply docs/gistStyleGuide.md:141]: I can't think of a good one. I think the text following "For example..." is pretty self-explanatory, so I'm going to resolve this.

Comment (rjyounes) [reply docs/gistStyleGuide.md:141]: This is a good point. I think I'll just remove that point and add a note about the validation annotations.

**Pull request merged**